### ISSUE TITLE
Possible to use custom dataset

### ISSUE THREAD

**Issue by shebna12** [zahid58/TwoStreamSepConvLSTM_ViolenceDetection#23]
Hi! May I know if it's possible to utilize your repo code to test for other datasets except hockey, movies, surv, rwf2000? Big thanks!

**Comment by zahid58** [comment]
Yes it's possible. But, I think you have to modify a lot of the dataset handling codes such as preprocessing, dataloader etc.
